exasol/analytics/query_handler/query/select.py

```diff
@@ -101,17 +104,19 @@ class ModifyQuery(CustomQuery, AuditData):
     def __init__(
         self,
         query_string: str,
-        # Using DBObjectName instead of str, because counting rows of the modified table
-        # requires to use fully_qualified
-        db_object_name: DBObjectName,
+        db_object_ref: DBObjectNameWithSchema,
```

Comment:
revert this change not all DBObjects have a Schema, for example Connection, if you want to find out if the Object has a schema, you need to do is instance `DBObjectNameWithSchema` in the Log Rewritter

Reply:
see next push 
* not supporting string values anymore but only objects `DbObjectType`, `DBObjectName`, `DbSpanType`
* using private attributes and public (read-only) properties 
* not updating `audit_fields`